### PR TITLE
Hw 2

### PR DESCRIPTION
**Pull request by Pranav-PSU**
01-flexbox: added solutions to the game
02-cssgrid: added solutions to the game
03.count.html: added bootstrap classes and HTML element to match the output with the screenshot
03-count.js: added code to highlight the input word in the text

04-background: added bootstrap classes and HTML elements to match the output with the screenshot
04-background.js: Added the code to change the background color

05-form.html: Created a form using HTML and Bootstrap which is matched with the screenshot.
05-form.js: Added code to get the values and print the output in the console.

External resources used: 
http://jsfiddle.net/5V8mV/1/ : for generating the background colors.
https://getbootstrap.com/docs: for bootstrap classes and components. 

Feedback needed:
I would appreciate any feedback on my implementation, coding standards, and formatting.

### CODE REVIEW

**Review by Rajashree-Pethkar — CHANGES_REQUESTED** [review]
Found some accessibility errors. And the border-radius is missing for form.

Screenshot 2023-04-27 at 10 33 09 PM
Screenshot 2023-04-27 at 10 33 32 PM

File: hw2/03-count.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  <body class="bg-dark">
+    <main>
+      <section>
+        <div class="container mt-5">
+          <div class="row justify-content-center">
+            <div class="col-sm-9 col-lg-5 col-md-9 bg-light p-3 rounded">
+              <h1>Exercise 03 - Count</h1>
+              <hr />
+              <div>
+                <label>Highlight all occurences of a word:</label>
```
Comment (rohan-singh1) [inline hw2/03-count.html]: for attribute missing

**Review by rohan-singh1 — APPROVED** [review]
Orphaned form label, but it has already been pointed out by Rajashree. Otherwise looks good.

File: hw2/03-count.html (hunk 1 above)
Comment (Pranav-PSU) [reply hw2/03-count.html]: fixed

**Comment by Pranav-PSU** [comment]
All suggested changes are done. Merging the pull request.

**Pull request merged**